README.md

```diff
@@ -12,7 +12,9 @@ import pasha as psh
 # Get some random input data
 inp = np.random.rand(100)
 
-# Allocate output array via EXtra-pasha.
+# Allocate output array via pasha. The returned array is
+# guaranteed to be accessible from any worker, and may
+# reside in shared memory.
 outp = psh.array(100)
```

Comment:
Not about the wording, but is it worth demoing `array_like` here, or passing dtype explicitly? Up to you, but relying on the default dtype tends to make me stop and think about what the default is.

Reply:
That makes sense, the first example should hit the mark.

However, as part of the pycalibration MR today it hit me that it might make sense to actually use the same API for array creation as numpy, i.e. `Context.array(array_like)`, `Context.zeros(shape, dtype)`, `empty`, `ones`, `xxx_like` and so on? 

This would not be part of this PR obviously, so let's postpone this.